config/initializers/country_names.rb

```diff
@@ -7,7 +7,7 @@ module Implementation
         include Base, Flatten
 
         def available_locales
-          I18nData.languages.keys.map(&:to_sym)
+          ::Spree::Store.current.available_locales.map(&:to_sym)
```

Comment:
current_store method is not initialized at this point so I had to use ::Spree::Store.current